libs/application/templates/criminal-record/src/fields/ExamplePaymentPendingField.tsx

```diff
@@ -0,0 +1,137 @@
+import React, { FC, useEffect, useState } from 'react'
+import { useQuery, gql, useMutation } from '@apollo/client'
+import {
+  CustomField,
+  FieldBaseProps,
+  formatText,
+  DefaultEvents,
+} from '@island.is/application/core'
+import { Box, Button, Text } from '@island.is/island-ui/core'
+import { m } from '../lib/messages'
+import { useLocale } from '@island.is/localization'
+import { SUBMIT_APPLICATION } from '@island.is/application/graphql'
+
+const QUERY = gql`
+  query status($applicationId: String!) {
+    applicationPaymentStatus(applicationId: $applicationId) {
+      fulfilled
+    }
+  }
+`
+
+interface Props extends FieldBaseProps {
+  field: CustomField
+}
+
+interface PaymentStatus {
+  fulfilled: boolean
+}
+
+export const ExamplePaymentPendingField: FC<Props> = ({
```

Comment:
Very minor but I noticed it still contains the `Example` prefix 🤓

Reply:
Fixing it